src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -37,11 +37,13 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
     expandedRow: style({
       backgroundColor: themeColor200,
       padding: Spacing.Medium,
-      width: '97%',
+      borderRadius: `0px 0px ${Spacing.Small}px ${Spacing.Small}px`,
```

Comment:
Note that the border-radius from ExpandableRow's style wasn't applied on the right element... That's the problem with too many nested elements :-)